client/modules/wj-css/src/components/_links.scss

```diff
@@ -0,0 +1,49 @@
+@use "sass:list";
+@use "sass:selector";
+
+a[href]:not([href*="javascript:;"]) {
+  // Base link styling.
+  text-decoration: currentColor underline 0.1em;
+  transition: color 0.05s, text-decoration-color 0.1s;
+
+  // TODO: replace hardcoded links with some other system
+  // for now this will work, but it's a little messy
+  // prettier-ignore
+  $internal-links-list:
+    '^="/"',
+    '^=".."',
+    '^="#"',
+    '*="wikijump.com"',
+    '*="wikijump.org"',
+    '*="wikijump.test"',
+    '*="wikijump.dev"',
+    '*="localhost"';
```

Comment:
I don't think we use `localhost` for anything.

More notably, is this won't work for any custom domains, and I don't want to have to commit and make a new `client` build to support it, it should be automatic.

Not a blocker for this PR since you have a TODO, but just a note.

Reply:
If you run a development server and don't use any tricks you'll run it on `localhost`. All of the `client/` web dev modes currently use `localhost`.

Also, yeah, totally aware of that issue. This particular bit of the CSS is kind of a placeholder until we figure out a solution. I think the solution most likely to work is probably environment variables.